src/main/webapp/app/pages/curation/collapsible/ReviewCollapsible.tsx

```diff
@@ -202,6 +228,55 @@ export const ReviewCollapsible = (props: IReviewCollapsibleProps) => {
     }
   };
 
+  const getMultiSelectionReviewContent = () => {
+    const multiSelectionReviewLevel = props.baseReviewLevel as MultiSelectionReviewLevel;
+    const joinedNewParts = [];
+    const joinedOldParts = [];
+    for (const reviewLevel of multiSelectionReviewLevel.getReviewLevels()) {
+      joinedNewParts.push(reviewLevel.currentVal);
+      joinedOldParts.push(reviewLevel.reviewInfo.lastReviewedString);
+    }
+    /* eslint-disable no-console */
+    console.log(joinedNewParts, joinedOldParts);
+    return (
+      <div className="mb-2">
+        <ReactDiffViewer
+          styles={REACT_DIFF_VIEWER_STYLES}
+          showDiffOnly
+          extraLinesSurroundingDiff={0}
+          oldValue={joinedOldParts.filter(p => p !== '').join('\t')}
+          newValue={joinedNewParts.filter(p => p !== '').join('\t')}
```

Comment:
I wonder whether tab is the best way to join or use comma.

Reply:
Comma looks better especially if we have more than two selections like for allele state
![image](https://github.com/oncokb/oncokb-transcript/assets/59149377/8ea392a3-b6b0-4216-8705-fe88bab3f246)